functorch/experimental/cond.py

```diff
@@ -151,6 +154,22 @@ def cond_python_dispatcher(*args):
     return cond(*args)
 
 
+@cond.py_impl(torch._C._functorch.TransformType.Functionalize)
+def cond_functionalize(interpreter, pred, true_fn, false_fn, inputs):
+    reapply_views = interpreter.functionalizeAddBackViews()
+    mode = 'mutations_and_views' if reapply_views else 'mutations'
+    unwrapped_inputs = _unwrap_all_tensors_from_functional(inputs, reapply_views=reapply_views)
+    functional_true_fn = make_fx(functionalize(true_fn, remove=mode))(*unwrapped_inputs)
+    functional_false_fn = make_fx(functionalize(false_fn, remove=mode))(*unwrapped_inputs)
```

Comment:
We shouldn't be applying the function `make_fx(functionalize(true_fn, remove=mode))` to some inputs (unwrapped_inputs) because then that returns a Tensor. We want functional_true_fn to be a function, right?